_guide_topics/grasshopper/hops-component.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Hops Component
+description: Hops is a component for Grasshopper that solves external Grasshopper documents.
+authors: ['steve_baer', 'scott_davidson']
+sdk: ['Grasshopper']
+languages: ['C#']
+platforms: ['Windows', 'Mac']
+categories: ['In Depth']
+origin:
+order: 2
+keywords: ['developer', 'grasshopper', 'components']
+layout: toc-guide-page
+---
+
+## Overview <img src="{{ site.baseurl }}/images/win-logo-small.png" alt="Windows" class="guide_icon"> 
+{: #hops }
+
+<img src="{{ site.baseurl }}/images/hops-overview.png">{: .img-center  width="100%"}
+
+Hops treats external Grasshopper documents as functions. This has the same effect on the structure of Grasshopper documents as functions in other programming languages do:
```

Comment:
Hops adds functions to Grasshopper. The functions are defined in separate Grasshopper documents. Like other programming languages, functions let you: